guides/common/attributes.adoc

```diff
@@ -135,8 +137,10 @@ ifeval::["{build}" == "foreman-deb"]
 :package-install: apt-get install
 :package-update: apt-get update
 :package-clean: apt-get clean
-:certs-generate: foreman-proxy-certs-generate
 :package-remove: apt-get remove
+:package-install-project: apt-get install
+:package-remove-project: apt-get remove
+:certs-generate: foreman-proxy-certs-generate
```

Comment:
This is a bit misleading. This simply doesn't exist on Debian and there is no equivalent.

Reply:
Elaborate on this, please. This is used to make the guides that build for Debian (those are the install guides only)  to render both the `package-install-project` and `package-install` attributes as `apt-get install`. And the same for remove.

Reply:
Specifically `foreman-proxy-certs-generate` does not exist. There is no CA tooling. The closest we get is using the Puppet CA but I don't think you want to write up instructions for that.

Reply:
I've opened the issue #213 to address that and other your feedback.